src/DIRAC/RequestManagementSystem/Agent/CleanReqDBAgent.py

```diff
@@ -148,12 +157,32 @@ def execute(self):
                 self.log.info("execute: deleting request '%s' with status %s" % (requestID, status))
                 delRequest = self.requestClient().deleteRequest(requestID)
                 if not delRequest["OK"]:
-                    self.log.error("execute: unable to delete request '%s': %s" % (requestID, delRequest["Message"]))
+                    self.log.error("execute: unable to delete request", "'%s': %s" % (requestID, delRequest["Message"]))
                     continue
                 deleted += 1
 
+        # optional: Set Scheduled requests to Cancelled if older than threshold
+        if self.cancelGraceDays > 0:
+            statusList = ["Scheduled"]
+            result = self.requestClient().getRequestIDsList(statusList, self.DEL_LIMIT)
```

Comment:
```suggestion
            result = self.requestClient().getRequestIDsList(["Scheduled"], self.DEL_LIMIT)
```
to avoid possible risks in re-definitions of `statusList`

Reply:
A redefinition between one line and the next ?!